h/models/organization.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+
+
```

Comment:
I'm not sure why lint isn't catching it but there should be only one blank line here